format_ideas/gaussian1d-option-4.yaml

```diff
@@ -0,0 +1,31 @@
+# Option 4: Inputs and outputs are lists instead of omap.  Does this better
+# express the fact that the inputs and outputs are ordered and not actually
+# keyed on name?
```

Comment:
In modeling, it can be either, referred to by name or position.

Reply:
This does seem to make more sense to me, inputs and outputs as lists.

Reply:
When models are combined with `CompoundModel`, is there a way to specify that output `foo` should be sent in as input `bar`?  Or does it always go by order?

Reply:
Always by order IIRC